documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -1,54 +1,125 @@
-  - Opening a PR will trigger a CI run that will build the release and publish it to GitHub as a draft
-  - Merging the PR will trigger a CI run that will publish the GitHub release (it will also create a git tag)
-
-**Post-Release**
+## ✅ Release Checklist
 
-- [ ] Open a pull request against the `master` branch with a merge of the `releases` branch. Conflict resolution should ignore the changes to `version.go` (keep the `-dev` version from master). Do NOT delete the `releases` branch when doing so!
-- [ ] Update [RELEASE_ISSUE_TEMPLATE.md](https://github.com/filecoin-project/lotus/blob/master/documentation/misc/RELEASE_ISSUE_TEMPLATE.md) with any improvements determined from this latest release iteration.
-- [ ] Create an issue using [RELEASE_ISSUE_TEMPLATE.md](https://github.com/filecoin-project/lotus/blob/master/documentation/misc/RELEASE_ISSUE_TEMPLATE.md) for the _next_ release.
+### Before RC1
+- [ ] (network upgrade) Make sure all [Lotus dependecies are updated to the correct versions for the network upgrade](https://github.com/filecoin-project/lotus/blob/master/documentation/misc/Update_Dependencies_Lotus.md)
+   - Link to PR:
+- [ ] Open PR against [RELEASE_ISSUE_TEMPLATE.md](https://github.com/filecoin-project/lotus/blob/master/documentation/misc/RELEASE_ISSUE_TEMPLATE.md) with title `docs(vX.Y.Z): release template improvements` for improving future releases.
+   - Link to PR:  
+   - This will get merged in a `Post Release` step, but improvements are better done by collecting notes along the way rather than just thinking about it at the end.
+- [ ] Fork a new branch (`release/vX.Y.Z` or `release/miner/vX.Y.Z`) from `master` and make any further release-related changes to this branch.
+- [ ] In the `master` branch, bump the version in `build/version.go`  to `vX.Y.(Z+1)-dev`. Run `make gen && make docsgen-cli` before committing changes.
+   - Link to PR: 
+
+### RCs
+
+[//]: # (Copy/paste this "RCX" section for each RC, and increment "X")
+#### RCX
+> [!IMPORTANT]
+> These PRs should be done in and target the `release/vX.Y.Z` or `release/miner/vX.Y.Z` branch.
+
+**Backport PR**
+- [ ] All explicitly tracked items from `Dependencies for releases` have landed
+- [ ] Backported [everything with the "backport" label](https://github.com/filecoin-project/lotus/issues?q=label%3Arelease%2Fbackport+) 
+- [ ] Removed the "backport" label from all backported PRs (no ["backport" issues](https://github.com/filecoin-project/lotus/issues?q=label%3Arelease%2Fbackport+))
+- [ ] Create a PR with title `build(vX.Y.Z): backport changes for vX.Y.Z-rcX`
+   - Link to PR: 
+- [ ] Merge PR 
+
+**Release PR**
+- [ ] Update the version string in `build/version.go` to one ending with '-rcX'. 
+    - Ensure to update the appropriate version string based on whether you are creating a node release (`NodeBuildVersion`), a miner release (`MinerBuildVersion`), or both.
+- [ ] Run `make gen && make docsgen-cli` to generate documentation
+- [ ] Changelog prep
```

Comment:
Is it right that this should be happening in `release/vX.Y.Z` rather than in `master` and then cherry picked.

Reply:
It is correct in the sense that this is what we have done previously. I do not have any particular stance on whether we should keep it the current way or change to land in master and then cherry-pick."

Reply:
Hmm, thinking on this more...

If we do the changelog updeates in release/vX.Y.Z then if someone looks at the the changelog in the master branch, they won't actually see the latest changelog for an in process release.   That's not ideal, but I assume we can live with it since the GitHub releases themsleves have the curated/accurate changelog text in the release notes.

If we do the changes to the changelog in master, then we need backport the changelog change when we backport other changes.  That potentially gets a bit messy if there are other changelog entries that have been added in the interim for the "unreleased version".

I don't have a strong thought here so maybe we just leave as-is currently and assess this next time through.